pkg/utils/nodeclaim/nodeclaim.go

```diff
@@ -485,3 +483,10 @@ func IsPastEmptinessTTL(nodeClaim *v1beta1.NodeClaim, clock clock.Clock, nodePoo
 	return nodeClaim.StatusConditions().GetCondition(v1beta1.NodeEmpty).IsTrue() &&
 		!clock.Now().Before(nodeClaim.StatusConditions().GetCondition(v1beta1.NodeEmpty).LastTransitionTime.Inner.Add(nodePool.Spec.Deprovisioning.EmptinessTTL.Duration))
 }
+
+func GetDriftedTime(nodeClaim *v1beta1.NodeClaim) time.Time {
```

Comment:
You can correct me if I'm wrong, but this shouldn't be needed anymore, since we just use it for deprovisioning.